src/lib/ggpo/platform_linux.h

```diff
@@ -8,20 +8,50 @@
 #ifndef _GGPO_LINUX_H_
 #define _GGPO_LINUX_H_
 
-#include <stdio.h>
-#include <stdarg.h>
+#include <sys/stat.h>
 #include <sys/types.h>
+#include <sys/time.h>
+#include <ctime>
+#include <sys/ioctl.h>
+#include <sys/socket.h>
 #include <unistd.h>
+#include <netinet/in.h>
+#include <stdio.h>
 #include <stdlib.h>
+#include <stdarg.h>
+#include <climits>
+#include <signal.h>
+#include <errno.h>
+#include <string.h>
+#include <arpa/inet.h>
+
+#include "pevents.h"
+
+#define DebugBreak() (raise(SIGTRAP))
```

Comment:
Ideally, none of the definitions from here to line 44 would be required, since they're specifically platform specific.